quadtree.js

```diff
@@ -151,4 +151,35 @@ class QuadTree {
 
     return found;
   }
-}
\ No newline at end of file
+
+  removeInRange(range){
+      if(!range.intersects(this.boundary)){
+          return false;
+      }
+
+      for(let i = this.points.length - 1; i >= 0 ; i--){
```

Comment:
Could this loop be better served using Array.filter?

Reply:
Yes, Thank you!
Iv'e added a filter instead of the for loop.